docs/getting-started/development.md

```diff
@@ -18,31 +22,32 @@ $ vagrant plugin install vagrant-group
 # Automatically manage Guest Additions versions
 $ vagrant plugin install vagrant-vbguest
 ```
-
-You'll need to structure your Moodle development environment appropriately. Ubiquitous sets up a synced folder for `../Moodle` relative to itself to get your source code to the application server:
-
-```
-.
-├── Moodle
-└── Ubiquitous
-```
-
 Start up all of the machines necessary for a testing environment:
 
 ```
+$ cd moodle-ubiquitous
 $ vagrant group up dev
 ```
 
-The first time you start the servers, and whenever you make changes to the Salt states, you'll need to apply the states to the machines:
+The [Vagrantfile](/Vagrantfile) makes assumptions about the host environment, including:
```

Comment:
This should be clear before you start running commands, hence the earlier directory structure. Please relocate up.